test/e2e/e2e_suite_test.go

```diff
@@ -59,12 +65,8 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	defer func() {
-		if !config.IsSoakTest && !config.IsUpgradeTest {
```

Comment:
I think we should keep the cleanup. At a later point when we document how to run this locally, users would want the cleanup to be done in their cluster.